srcpkgs/tang/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'tang'
+pkgname=tang
+version=8
+revision=1
+build_style=meson
+hostmakedepends="asciidoc pkg-config"
+makedepends="http-parser-devel jansson-devel libjose-devel openssl-devel zlib-devel"
+depends="xinetd"
+make_dirs="/var/db/tang 0700 tang tang"
+short_desc="Tang binding daemon"
+maintainer="Jiri Belka <jirib79@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/latchset/tang"
+distfiles="https://github.com/latchset/tang/releases/download/v${version}/tang-${version}.tar.xz"
+checksum=03958bf27c24aa9892f6138d287d39bcc149e7ad72163c27b529aa3c6b066ede
+
+system_accounts="$pkgname"
```

Comment:
We generally start new usernames with `_` to avoid conflicts in case users already use the specific name.